src/js/messaging/containers/Folder.jsx

```diff
@@ -312,9 +313,12 @@ export class Folder extends React.Component {
             onToggleMoveTo={() => this.props.toggleFolderMoveTo(id)}/>
       );
 
+      const attachmentIcon = message.attachment ? (<span className="fa fa-paperclip" aria-label="Message has an attachment"></span>) : null;
```

Comment:
If it works, that's fine, but I usually see `fa` used with `i` elements. Don't know if there might be other styles that depend on it being an `i` element. Could also wrap it (if turning it into `i`) and have the `"Message has an attachment"` inside a `usa-sr-only` classed `span`.